UI/window-basic-settings.cpp

```diff
@@ -4671,16 +4671,16 @@ void OBSBasicSettings::SpeakerLayoutChanged(int idx)
 		 */
 		PopulateSimpleBitrates(ui->simpleOutputABitrate, isOpus);
 
-		const char *encoder_id = QT_TO_UTF8(
+		string encoder_id = QT_TO_UTF8(
 			ui->advOutAEncoder->currentData().toString());
```

Comment:
You could just replace the macro here with `.toStdString()` (internally it does pretty much the same), or keep it. a QString and wrap the parameters below themselves in `QT_TO_UTF8`. The latter would probably look nicer since `rec_encoder_id` already is, but it is technically probably less efficient.

Reply:
It might be worth doing this across the entire codebase as part of another PR, we mostly use `QT_TO_UTF8` for `QString` -> `string` but there are places where we also use `toStdString`.

Reply:
Changed to use `toStdString` for both to avoid use of `QT_TO_UTF8` entirely.